book/quick-brick/tvOS/Plugins/PlayerDependants/intro.md

```diff
@@ -0,0 +1,120 @@
+## TvOS Video Player Depandant plugins
+
+### Introduction
+This Document will explain general structure how Video Player Depandant plugin are working on TvOS with QuickBrick
+
+#### Supported Apple TV variant:
+Apple TV 4th Generation or later using tvOS 11.0 or higher
+
+#### Available from ZappSDK 10.1.1 and above
+
+#### Required Applicaster Frameworks:
+
+* ZappPlugins
+
+### Content
+* <a href="#general">General</a>
+* <a href="#api">API</a>
+* [How To Create New Plugin?](/quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
+
+### Plugins
+
+* [Google Ineractive media Ads](/quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
+
+### Dependant Player types
+* `advertismentPlaying` - Video ads plugins type
+
+<a name="general" />
+
+### General
+
+Video Player Depandant plugins, plugin type that require player instance to the invocation. Example of such plugins can be GoogleIMA or Analytics for the player.
+
+To implement type of such plugins need implemented on player plugin side two protocols:
+`QBPlayerProtocol` - This protocol define what var and what func can be retrieved from the player
+`ZPDependablePlayerPluginProtocol` - Implementation of this protocol will defined what types of dependant plugin player will support
+
+On Player dependant plugins needed to be implement two protocols
+`ZPPlayerDependantPluginProtocol` - This protocol defines that plugin can get player instance to retieve data
+`QBPlayerObserverProtocol` - By this protocol player will send event to plugin that dependant by player
```

Comment:
Contains implementation that allows the player to send events to the dependent plugins.